packages/kui-builder/lib/configure.js

```diff
@@ -49,6 +49,40 @@ const readIndex = options =>
     })
   })
 
+/**
+ * Unify the given settings of configurations
+ *
+ */
+const unify = (setting1, setting2) => {
+  const uniqe = arr => {
+    return arr.filter((element, index) => arr.indexOf(element) === index)
+  }
+
+  const unifiedSettings = setting1
+
+  for (const key in setting2) {
+    if (!unifiedSettings[key]) {
+      unifiedSettings[key] = setting2[key]
+    } else {
+      if (Array.isArray(unifiedSettings[key]) && Array.isArray(setting2[key])) {
+        // concatenate the arrays
+        unifiedSettings[key] = unifiedSettings[key].concat(setting2[key])
+        // remove the duplicated elements and join the array into a string
+        unifiedSettings[key] = uniqe(unifiedSettings[key]).join(' ')
+      } else if (!Array.isArray(unifiedSettings[key]) && Array.isArray(setting2[key])) {
```

Comment:
At some point, webpack-watch will read bodyCss as `bodyCss:'not-electron kui'` (maybe from existing index.html?), so I have to add the case here.